### PR TITLE
[MRG] ADD:  Brain.remove_foci()

### PR DESCRIPTION
**Pull request by christianbrodbeck**
Seemed like it was missing and simple enough to add.

### CODE REVIEW

**Comment by christianbrodbeck** [comment]
Yes, `remove_data()` and `remove_labels()`, glad you asked :)

**Comment by christianbrodbeck** [comment]
We could add a new `Brain` attribute to keep track of unnamed foci too, but it would add some more overhead...

File: surfer/viz.py [hunk 1]
```diff
@@ -1454,6 +1454,34 @@ def remove_data(self, hemi=None):
         if all(len(brain.data) == 0 for brain in self.brains):
             self.n_times = self._times = None
 
+    def remove_foci(self, foci=None):
```
Comment (agramfort) [inline surfer/viz.py]: please call it name to be consistent with add_foci API.

File: surfer/viz.py [hunk 2]
```diff
@@ -1454,6 +1454,34 @@ def remove_data(self, hemi=None):
...
+        """Remove foci added with ``Brain.add_foci()``.
+
+        Parameters
+        ----------
+        foci : str | Sequence of str
```
Comment (agramfort) [inline surfer/viz.py]: str | list of str

Comment (agramfort) [reply surfer/viz.py]: or better:

str | list of str | None

File: surfer/viz.py [hunk 3]
```diff
@@ -1454,6 +1454,34 @@ def remove_data(self, hemi=None):
...
+            Names of the foci to remove (default all).
```
Comment (agramfort) [inline surfer/viz.py]: If None, remove all focis.

**Comment by larsoner** [comment]
Thanks @christianbrodbeck

**Pull request merged**